python/tests/test_adjoint_solver.py

```diff
@@ -158,7 +164,8 @@ def J(mode_mon):
         objective_arguments = obj_list,
         design_regions = [matgrid_region],
         frequencies=frequencies,
-        decay_fields=[mp.Ez])
+        decay_fields=[mp.Ez],
+        decimation_factor=10)
```

Comment:
Why are some default values for `decimation_factor` 5, some 10, and others 1? Shouldn't they all have the same value (either 1 or a value determined by the sources/frequencies requested)?